pkgs/tools/video/nordlicht/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, ffmpeg, popt, help2man }:
+    with cmake;
+    stdenv.mkDerivation rec {
```

move indention of the entire file over to the beginning please
```suggestion
stdenv.mkDerivation rec {
```